### ISSUE TITLE
Author page: image tag is incorrect, it takes the cover image.

### ISSUE THREAD

**Issue by fabienwang** [TryGhost/Ghost#6221]
Hello, i have an issue since few months on author page.
I can't get author image & cover,
the author image is always the same as the cover image while i use both tags.

![author_page](https://cloud.githubusercontent.com/assets/8741873/11808204/45718f88-a31f-11e5-882e-ff56d7bd7bb8.jpg)

please have a look at the Ghost-Flat theme, specifically the part: https://github.com/fabienwang/Ghost-Flat/blob/master/partials/head-image.hbs#L16

**Comment by ErisDS** [comment]
Hi @fabienwang, this is a pretty impressive catch.

There is some code in the `{{ghost_head}}` helper which overwrites `image` with `cover`. It's only supposed to have an effect locally for the meta data, but because objects in JS are passed by reference this ends up overwriting the real data.

You may have noticed this doesn't affect Casper, and indeed it would not affect the majority of themes. It specifically effects your theme because you're rendering the header via a partial which is called directly from the default/default-wide layout file, rather than from a template file.

As `{{{body}}}` gets executed first, if this code had lived inside the template file, or you had been using content blocks to define the block in the layout, and the content inside the template, this wouldn't have happened as the data wouldn't have been modified until after your template had rendered.

In short, this is a very, very obscure bug that will only affect the image property of the author on author pages where the `{{author.image}}` property is called after `{{ghost_head}}`.

PR with a fix is incoming.

**Comment by fabienwang** [comment]
Damn you're fast. Thanks @ErisDS

**Comment by ErisDS** [comment]
No problem, don't leave it months next time :wink:  Don't forget you can always drop a note in the #themes channel in slack as well if you're having trouble.
